package.json

```diff
@@ -46,6 +50,7 @@
     "query-string": "^5.1.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-focus-on": "^3.5.0",
```

Comment:
Do we need this as a dep? I think this is leftover from the Paragon issues in CI.

Reply:
I couldn't get tests to pass without it as a dep, so yes.